src/test/resources/WithdrawPageTestCases

```diff
@@ -0,0 +1,72 @@
+1. Withdraw transaction happy flow
+Pre-condition:
+- Navigate to Puppy Bank login page.
+- Login as admin.
+- Automatic transition to the dashboard page occurs.
```

Comment:
Expected results should not be stored in Pre-condition section. Normally this section either says something like: 
- User is on Some Page
- User is logged as Admin
Or you can list actions here like: 
- Login as admin 
- Navigate to some page
- Create Transaction